INSTALL.md

```diff
@@ -53,7 +55,7 @@ Run the following commands:
 
 ```bash
 sudo dnf install make g++ git autoconf
-sudo dnf install zlib-devel libpng-devel libXpm-devel libX11-devel libXft-devel libXinerama-devel fontconfig-devel libXext-devel libXrender-devel libXfixes-devel
+sudo dnf install zlib-devel libpng-devel libXpm-devel libX11-devel libXft-devel libXinerama-devel fontconfig-devel libXext-devel libXrender-devel libXfixes-devel libcairo2-devel libpango1.0-devel
```

Comment:
I didn't actually test this on Fedora, I just followed the convention of changing "dev" to "devel"...